### PR TITLE
Add Sentinel 0.14.1

### PR DESCRIPTION
**Pull request by petems**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/sentinel.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'sentinel' do
+  SENTINEL_VERSION = '0.14.1'.freeze
```
Comment (vitorgalvao) [inline Casks/sentinel.rb]: Please don’t do it like this. `version` can be interpolated just fine. Look at any other cask as reference; it’s a common technique.

File: Casks/sentinel.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  SENTINEL_SHA256  = '8a6f714850fb30afac86cc73f653add96168d648fa3110072185fe2b5379e143'.freeze
+
+  version SENTINEL_VERSION
+  sha256 SENTINEL_SHA256
+
+  url "https://releases.hashicorp.com/sentinel/#{SENTINEL_VERSION}/sentinel_#{SENTINEL_VERSION}_darwin_amd64.zip"
+  name 'sentinel'
+  homepage 'https://docs.hashicorp.com/sentinel/'
+
+  binary 'sentinel', target: 'sentinel'
```
Comment (vitorgalvao) [inline Casks/sentinel.rb]: `target` is doing nothing here.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Is this open-source?

**Comment by petems** [comment]
> Is this open-source?

No, only as a binary.

File: Casks/sentinel.rb (hunk 1 above)
Comment (petems) [reply Casks/sentinel.rb]: Ok, fixed! 👍

File: Casks/sentinel.rb (hunk 2 above)
Comment (petems) [reply Casks/sentinel.rb]: Ok, fixed! 👍

File: Casks/sentinel.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+cask 'sentinel' do
+  version '0.14.1'
+  sha256 '8a6f714850fb30afac86cc73f653add96168d648fa3110072185fe2b5379e143'
+
+  url "https://releases.hashicorp.com/sentinel/#{version}/sentinel_#{version}_darwin_amd64.zip"
```
Comment (vitorgalvao) [inline Casks/sentinel.rb]: Please add an [`appcast`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md): `https://docs.hashicorp.com/sentinel/downloads/`

Also, leave on the box that allows edits from maintainers, as that speeds up the process.

File: Casks/sentinel.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+cask 'sentinel' do
+  version '0.14.1'
```
Comment (vitorgalvao) [inline Casks/sentinel.rb]: It’s at `0.14.2`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/sentinel.rb [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+  sha256 '8a6f714850fb30afac86cc73f653add96168d648fa3110072185fe2b5379e143'
+
+  url "https://releases.hashicorp.com/sentinel/#{version}/sentinel_#{version}_darwin_amd64.zip"
+  name 'sentinel'
```
Comment (vitorgalvao) [inline Casks/sentinel.rb]: `Sentinel`

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/sentinel.rb (hunk 4 above)
Comment (petems) [reply Casks/sentinel.rb]: Hah that must've been released recently, it was 0.14.1 this morning

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**